sklearnex/tests/test_n_jobs_support.py

```diff
@@ -36,6 +36,7 @@
 X, Y = make_classification(n_samples=40, n_features=4, random_state=42)
 
 
+@pytest.mark.allow_sklearn_fallback
```

Comment:
Test of n_jobs support in sklearnex estimators is not expected to fall back on any patched method.

Reply:
And yet it does. LogisticRegression predict_proba and predict_log_proba silently fall back to sklearn in daal4py here: https://github.com/intel/scikit-learn-intelex/blob/main/daal4py/sklearn/linear_model/logistic_path.py#L820